ui/objects/semantic-ui/js/objects/search/components/ResultsListItem.jsx

```diff
@@ -130,10 +132,7 @@ export const ResultsListItemComponent = ({ result, appName }) => {
           <ItemExtra>
             <Label size="large">
               Vloženo:{" "}
-              {created.substring(
-                created.indexOf(",") + 1,
-                created.lastIndexOf(",")
-              )}{" "}
+              {creationDate}{" "}
```

Comment:
Can we rather display human readable / internationalized date here instead of machine ISO timestamp?